WordPress-VIP-Go/ruleset.xml

```diff
@@ -9,63 +9,63 @@
 	<!-- Things that may be incompatible with the VIP Go infrastructure and needs a dev to review -->
 	<rule ref="WordPressVIPMinimum.Functions.RestrictedFunctions.file_ops_delete">
```

Comment:
@hanifn I think each of the rules with amended messages here, need a `<type>warning</type>` element added.